packages/core/src/components/Tooltip/__stories__/tooltip.stories.js

```diff
@@ -31,144 +31,127 @@ const tooltipTemplate = args => {
   render: tooltipTemplate.bind({}),
   name: "Overview",
-
   args: {
     shouldShowOnMount: true,
     content: "I'm a tooltip"
   },
-
   parameters: {
     chromatic: {
       pauseAnimationAtEnd: true
     }
   }
 };
 
-export const Positions = {
-  // The modifier's purpose is to prevent the tipseen from being displayed when the user scrolls the story upwards / downwards.
-  render:
-    // Therefore, there is no need to move this prop in your implementations.
-    () => {
-      return (
-        <>
-          <div className="monday-storybook-tooltip_top">
-            <Tooltip
-              modifiers={modifiers}
-              content="Top"
-              shouldShowOnMount
-              position={Tooltip.positions.BOTTOM}
-              animationType={Tooltip.animationTypes.EXPAND}
-              withMaxWidth
-            >
-              <div />
-            </Tooltip>
-          </div>
-          <div className="monday-storybook-tooltip_bottom">
-            <Tooltip modifiers={modifiers} content="Bottom" shouldShowOnMount animationType="expand" maxWidth>
-              <div />
-            </Tooltip>
-          </div>
-          <div className="monday-storybook-tooltip_right">
-            <Tooltip
-              modifiers={modifiers}
-              content="Left"
-              position={Tooltip.positions.RIGHT}
-              shouldShowOnMount
-              animationType={Tooltip.animationTypes.EXPAND}
-              withMaxWidth
-            >
-              <div />
-            </Tooltip>
-          </div>
-          <div className="monday-storybook-tooltip_left">
-            <Tooltip
-              modifiers={modifiers}
-              content="Right"
-              position={Tooltip.positions.LEFT}
-              shouldShowOnMount
-              animationType={Tooltip.animationTypes.EXPAND}
-              withMaxWidth
-            >
-              <div />
-            </Tooltip>
-          </div>
-        </>
-      );
-    },
+export const TooltipWithTitle = () => (
+  <div className="monday-storybook-tooltip_title">
+    <Tooltip
+      content="Hidden columns"
+      title="Tooltip title"
+      withMaxWidth
+      shouldShowOnMount
+      position={Tooltip.positions.RIGHT}
+      modifiers={modifiers}
+    >
+      <div />
+    </Tooltip>
+  </div>
+);
+
+export const TooltipWitImage = () => (
+  <div className="monday-storybook-tooltip_image">
+    <Tooltip
+      content="Max width tooltip with a long text example"
+      title="Tooltip title"
+      withMaxWidth
+      shouldShowOnMount
+      image={image}
+      position={Tooltip.positions.RIGHT}
+      modifiers={modifiers}
+    >
+      <div />
+    </Tooltip>
+  </div>
+);
 
+export const Positions = {
+  render: () => {
+    return (
+      <div className="monday-storybook-tooltip_positions">
+        <div>
+          {/* The modifier's purpose is to prevent the tipseen from being displayed when the user scrolls the story upwards / downwards.
```

Comment:
We need to find a way to clean up stories

Reply:
you mean the section that is being rendered in the "source" area?

Reply:
yes, because it's irrelevant for users who copy the code to use it

Reply:
LiveEdit should solve I believe